models/request.js

```diff
@@ -17,20 +17,25 @@
  */
 
 module.exports = function(sequelize, DataTypes) {
-    return sequelize.define('Request', {
-	name: DataTypes.TEXT,
-	address: DataTypes.TEXT,
-        sms_raw_address: DataTypes.TEXT,
-	assigned_rc_region: DataTypes.TEXT,
-	city: DataTypes.TEXT,
-	state: DataTypes.TEXT,
-	zip: DataTypes.TEXT,
-        sms_raw_zip: DataTypes.TEXT,
-	phone: DataTypes.TEXT,
-        sms_raw_phone: DataTypes.TEXT,
-	email: DataTypes.TEXT,
-        source: DataTypes.TEXT,
-        serial: { type: DataTypes.TEXT, unique: true },
-        status: DataTypes.TEXT
-    })
-}
+  return sequelize.define('Request', {
```

Comment:
FYI, we prefer to minimize whitespace changes to make the diff easier to read.  I see you just added the lat/long columns, though.